lib/resources/dstu2/appointment.yaml

```diff
@@ -44,7 +50,7 @@ fields:
   example: |
```

Comment:
Not sure I'm following this description (I think @dennispatterson noticed this too): "If provided, then no schedule and start/end values MUST match the ones from the slot" ??

Reply:
That is taken directly from the FHIR spec. I make @torresdrew explain it to me all the time because I'm not still not convinced it's cromulent.

http://hl7.org/fhir/DSTU2/appointment-definitions.html#Appointment.slot

I'd be down for reducing it to just "The slot that this appointment is filling"

Reply:
How 'bout "The slot that this appointment is filling.  The start/end values MUST be the same as from the slot."

Reply:
I removed that portion because it doesn't actually apply with our implementation. The start and end times are defined entirely by the Millennium appointment and slot types builds and can't be overridden through the FHIR API. That could change in the future, but today everything is done via the slot id only

Reply:
Gotcha.  +1